src/RpcConfig.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Thesis\Amqp;
+
+use Thesis\Time\TimeSpan;
+
+/**
+ * @api
+ */
+final readonly class RpcConfig
+{
+    private const int DEFAULT_TIMEOUT = 2;
+
+    public TimeSpan $timeout;
+
+    /**
+     * @param ?non-empty-string $replyTo temporary queue name, must be unique for all instances of the application
```

Comment:
This cannot be achieved in Symfony DI, because it is compiled. Let's use a `callable(): non-empty-string` generator, defaulting to `static fn (): string => 'thesis.rpc.' . random()`. I suggest `callable` type, so that one can simply use an invokable DI service for that.